spec/template.dd

```diff
@@ -5,7 +5,7 @@ $(SPEC_S Templates,
 $(HEADERNAV_TOC)
 
 $(BLOCKQUOTE_BY Richard Deyman,
-I think that I can safely say that nobody understands C++ template mechanics.)
+I think that I can safely say that nobody understands template mechanics.)
```

Comment:
Since this one is a quote you might want to leave this one.

Reply:
This looks more like a "joke quote" based on the famous:

> I think I can safely say that nobody understands quantum mechanics.

-- Richard Feynman

http://www-history.mcs.st-andrews.ac.uk/Quotations/Feynman.html

Reply:
I'd say just remove.